caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -709,14 +751,22 @@ def fit_online(self, **kwargs):
                 additional parameters used to modify self.params.online']
                 see options.['online'] for details
 
+            estimates: caiman.source_extraction.cnmf.Estimates
+                Initialization can be passed as input: the initializer must be create by the same fit_online function
+
+            save_init_file: str
+                Path to the HDF5 file that will contain the initialization to be reused in the future
+
         Returns:
             self (results of caiman online)
         """
-        
+
```

Comment:
@agiovann If you initialize with an `estimates` object the `init_batch` variable should be determined by the estimates, e.g. it should not be larger than the length of `C` inside estimates (it could be smaller, ideally they are the same).